src/request-pipeline/context.js

```diff
@@ -227,6 +227,11 @@ export default class RequestPipelineContext {
         this.res.end();
     }
 
+    isNotModified () {
+        return this.req.method === 'GET' && this.destRes.statusCode === 304 &&
```

Comment:
It is used in two places. So we can remake this code as property `RequestPipelineContext.isNotModified`.